webapp/client/src/app/dashboard/search-flight/search-flight.component.ts

```diff
@@ -27,14 +27,13 @@ export class SearchFlightComponent implements OnInit {
     private formBuilder: FormBuilder) { }
 
   ngOnInit() {
-
     this.searchFlightForm = this.formBuilder.group({
       departingFrom: [null, [Validators.required]],
       arrivingAt: [null],
       departureDate: [null, [Validators.required]],
       arrivalDate: [null, [Validators.required]],
       passengerNumber: [1, [Validators.required]],
-      cabinClass: [CabinClass.ECONOMY.toString(), [Validators.required]],
+      cabinClass: [this.cabinsClassOptions[0], [Validators.required]],
```

Comment:
emmm, are you sure about that?

Reply:
you can't access enum's value like it was previously (cabinClass.ECONOMY.toString())
And i saw you created the 'cabinsClassOptions' that will transform the enum to an array